dune/grid/common/p2pcommunicator.hh

```diff
@@ -52,7 +52,7 @@ namespace Dune
     typedef std::vector< char >  BufferType;
 
     mutable BufferType buffer_;
-    const double factor_;
```

Comment:
no, I'm not sure. maybe it could be fixed another way. The problem is that GCC 4.4 currently spits out the following warning:

```
/data/and/ctest/gcc-44/src/dune-cornerpoint/dune/grid/common/p2pcommunicator.hh:51: error: non-static const member ‘const double Dune::SimpleMessageBuffer::factor_’, can't use default assignment operator
```

(note that I also think this is a weird one, probably it's a GCC 4.4 bug.)